### PR TITLE
Sync metal3-io master

### PR DESCRIPTION
**Pull request by elfosardo**

### CODE REVIEW

**Comment by derekhiggins** [comment]
ironic-conductor can't start, iirc ibmc is upstream only
> 2021-09-24 11:29:09.962 1 ERROR oslo_service.service ironic.common.exception.DriverLoadError: Driver, hardware type or interface ibmc could not be loaded. Reason: Unable to import the python-ibmcclient library.

**Comment by iurygregory** [comment]
> ironic-conductor can't start, iirc ibmc is upstream only
> 
> > 2021-09-24 11:29:09.962 1 ERROR oslo_service.service ironic.common.exception.DriverLoadError: Driver, hardware type or interface ibmc could not be loaded. Reason: Unable to import the python-ibmcclient library.

Yeah I agree with Derek, we don't support this driver downstream.

File: main-packages-list.ocp [hunk 1]
```diff
@@ -43,6 +43,7 @@ python3-packaging >= 20.4-1.el8ost
 python3-paste >= 3.2.4-1.el8ost
 python3-paste-deploy >= 2.0.1-4.el8ost
 python3-pint >= 0.10.1-1.el8ost
+python3-proliantutils
```
Comment (LorbusChris) [inline main-packages-list.ocp:46]: This probably should have been added to the OKD pkgs list as well?

Comment (dtantsur) [reply main-packages-list.ocp:46]: ++ we really need an okd job so that we stop doing this mistake

Comment (elfosardo) [reply main-packages-list.ocp:46]: d'oh! sorry, you're right :/

**Pull request merged**